code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -356,8 +360,10 @@
 	datum_reference.abno_radio = TRUE
 
 /mob/living/simple_animal/hostile/abnormality/proc/IsContained() //Are you in a cell and currently contained?? If so stop.
-	if(status_flags & GODMODE)
+//Contained also checks if 1. qliphoth exists and 2. if qliphoth is not zero. Needed for when breached abnorms get set to godmode
+	if((status_flags & GODMODE) && (!datum_reference.qliphoth_meter_max || datum_reference.qliphoth_meter))
```

Comment:
This will cause unintended behavior with abnos like KoD, who do not have qliphoth counter but breach.